docs/shipping/Containers/control-plane.md

```diff
@@ -0,0 +1,87 @@
+---
+id: Control-plane
+title: Control Plane
+overview: Control Plane is a hybrid platform that integrates multiple cloud services, such as AWS, GCP, and Azure, providing a unified and flexible environment for developers to build and manage backend applications and services across various public and private clouds.
+product: ['logs']
+os: ['windows', 'linux']
+filters: ['Containers', 'Most Popular']
+logo: https://logzbucket.s3.eu-west-1.amazonaws.com/logz-docs/shipper-logos/control-plane.png
+logs_dashboards: []
+logs_alerts: []
+logs2metrics: []
+metrics_dashboards: []
+metrics_alerts: []
+drop_filter: []
+---
+
+## Logs
+
+Control Plane is a hybrid platform that integrates multiple cloud services, such as AWS, GCP, and Azure, providing a unified and flexible environment for developers to build and manage backend applications and services across various public and private clouds.
+
+Deploy this integration to ship all `Org` logs from your Control Plane account to Logz.io.
+
+**Before you begin, you'll need**:
+
+* Control Plane account
+
+
+
+### Store the Logz.io secret as an Opaque Secret
+
+1. Log in to [Control Plane Console UI](https://console.cpln.io/).
+2. Navigate to **Secrets**.
+3. Create a new secret.
+4. Give the secret a name.
+5. Select `Opaque` from the **Secret Type list**.
+6. Paste `<<LOG-SHIPPING-TOKEN>>` into the content text box.
+7. Click **Create**.
+
+{@include: ../../_include/log-shipping/log-shipping-token.html}
+
+### Enable Control Plane logging
+
+#### Using the UI Console
+
+1. Open the [Control Plane Console UI](https://console.cpln.io/).
+2. Click on **Org** in the left menu.
+3. Click **External Logs** in the middle context menu.
+4. Select `Logz.io` and fill out the required fields.
+5. Select the Opaque secret created to authenticate to Logz.io.
+6. Click **Save**.
+
+#### Using the CLI
+
+1. Prepare the YAML manifest:
+
+```yaml
+kind: org
+name: ORG_NAME
+spec:
+  logging:
+    logzio:
+      credentials: //secret/<<LOG-SHIPPING-TOKEN>>
```

Comment:
Is this right? If we stored the shipping token as a secret, I don't think that you're expected to write it explicitly here, I would assume that you'd need to write the name of the opaque secret you created in the first step.
Can you please check this part?

Reply:
Correct, it's the secret's name that we need to specify, not the token!